crates/security/src/lib.rs

```diff
@@ -65,8 +65,9 @@ pub mod pallet {
 
     #[pallet::error]
     pub enum Error<T> {
+        /// Parachain is not running.
         ParachainNotRunning,
-        ParachainShutdown,
+        /// Arithmetic overflow.
         ArithmeticOverflow,
```

Comment:
Unsure if we should use `sp_runtime::ArithmeticError::Overflow` instead of this. See: https://github.com/interlay/interbtc/pull/527#discussion_r823625240

Reply:
You made a good point on the linked comment, probably we should go for consistency then, i.e. use `sp_runtime::ArithmeticError::Overflow` everywhere